src/tools/rest-frontend/resources/assets/html/index.html.in

```diff
@@ -22,6 +22,9 @@
         <!-- RSS Info -->
         <link rel="alternate" type="application/atom+xml" href="news/feed.rss" title="Elektra RSS Feed">
 
+        <!-- Fonts -->
+        <link href="https://fonts.googleapis.com/css?family=Lato|Libre+Franklin" rel="stylesheet">
```

Comment:
Please do not use cross-domain requests but copy the font into the repo (As done for other fonts already.).

Maybe some fonts are now obsolete and can be removed?

Reply:
Sure, this was intended for testing the fonts. I would convert and integtrate the fonts when we agreed upon them.